server/groups.py

```diff
@@ -0,0 +1,15 @@
+import requests
+from flask import request
+
+
+def get_invites_for_account(account):
```

Comment:
Let's add documentation for exactly what this function does and what it returns.